postgres/client.js

```diff
@@ -43,12 +64,28 @@ function query(query) {
  * @return {Promise}
  */
 function get(key) {
-  return client.query(GET(key))
+  const { schema, table } = findSchemaAndTable(key);
+
+  if (schema) {
+    return knex.withSchema(schema)
```

Comment:
I think it would be valuable to change your knex usage slightly to utilize a helper like:

```js
const baseQuery = (key) => {
  const { schema, table } = findSchemaAndTable(key);
  return schema
    ? knex(table).withSchema(schema)
    : knex(table);
};
```

Reply:
For sure, will handle now